cluster-scope/overlays/nerc-ocp-obs/kustomization.yaml

```diff
@@ -0,0 +1,73 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+commonLabels:
+  nerc.mghpcc.org/kustomized: "true"
+
+resources:
+- ../common
+- ../../bundles/node-feature-discovery
+- ../../bundles/patch-operator
+# - ../../bundles/rhods-operator
```

Comment:
Please do not leave commented-out items in these files. If we need them we'll add them in a future pull request.

Reply:
good point. agreed.

Reply:
good point. agreed.